lib/byron/test/data/cardano-node/node.config

```diff
@@ -12,7 +12,6 @@ NodeId:
 NumCoreNodes: 1
 PBftSignatureThreshold: 1
 Protocol: RealPBFT
-GenesisFile: test/data/cardano-node/genesis.json
```

Comment:
this one will be determined upon creation in `/tmp` knowing the location of `genesis.json` in `tmp`